QisFIOStreamExample.py

```diff
@@ -167,84 +170,79 @@ def qis_stream_and_FIO_example(module, testDirectory, streamDirectory):
     else:
         print("\tStream ran correctly")
 
-    # Stop the stream.  This function is blocking and will wait until all remaining data has
-    # been downloaded from the module
+    # Stop the stream.  This function is blocking and will wait until all remaining data has been downloaded from the module
     module.stopStream()
     time.sleep(1)
 
     print("Merging QIS and FIO data into one csv file.")
-
-    merge_file_location = merge_fio_qis_stream(
+    merge_file_location =  merge_fio_qis_stream(
         qis_stream_file=streamFileName,
         fio_output_file=fIOOutputPath[1:-1],
         unix_stream_start_time=str(streamStartTime)+"nS",
         rounding_option="round",  # or "insert",
         output_file=streamFileName.replace(".csv","")+"_merged_with_fio.csv"
     )
-    print("Merge file can be found at: " + merge_file_location)
 
+    # Here we use quarchpy display table function to nicely output the location of the 3 data file for the user to look at.
+    my_list2d = [["Qis Data", qisFilePath], ["FIO Data", fIOOutputPath[1:-1]], ["Merged Data", merge_file_location]]

```

Comment:
```suggestion
    file_paths = [["Qis Data", qisFilePath], ["FIO Data", fIOOutputPath[1:-1]], ["Merged Data", merge_file_location]]

```
Recommend having a better variable name